modulefiles/module_base.wcoss_dell_p3.lua

```diff
@@ -38,7 +38,7 @@ load(pathJoin("fms", "2021.03"))
 
 load(pathJoin("bacio", "2.4.1"))
 load(pathJoin("g2", "3.4.2"))
-load(pathJoin("g2tmpl", "1.10.1"))
+load(pathJoin("g2tmpl", "1.10.0"))
```

Comment:
This should be changed for all the machines.

Reply:
Done and committed at dac534d.